css/grayscale.scss

```diff
@@ -159,74 +159,60 @@ a {
 }
 
 .intro {
-  display: table;
+  display: flex;
+  position: relative;
+  flex-direction: column;
   width: 100%;
-  height: auto;
+  height: 100svh;
+  padding: 100px 0;
   text-align: center;
-  padding-top: 20rem;
   color: $font-color;
-  background: url("{{ site.baseurl }}/img/intro-bg.jpg") no-repeat center center;
-  background-size: cover;
+  background-color: $primary-color;
   -webkit-background-size: cover;
   -moz-background-size: cover;
+  background-size: cover;
   -o-background-size: cover;
+  background-attachment: scroll;
+  background-image: url('{{ site.baseurl }}/img/intro-bg.jpg');
+  background-position: center center;
+  background-repeat: none;
 
-  .intro-body {
-    display: table-cell;
-    vertical-align: middle;
-    position: relative;
-    z-index: 2;
-    .brand-heading {
-      font-size: 40px;
-    }
-    .intro-text {
-      font-size: 1.3rem;
-    }
+  .intro-text {
+    font-size: 2.3rem;
+    font-weight: 900;
```

Comment:
Let's bring the font size back to 1.3rem and remove the shadow, because it feels a bit out of place with the rest of the theme 😅 

During the latest upgrade, I spend a lot of time to optimize the font sizes for readability, so I prefer to keep them as is for now, and anyone who wants different sizes they can change them in their fork 😎